static/app/utils/useApiRequests.tsx

```diff
@@ -112,34 +112,19 @@ type Options = {
    * "reloading" states
    */
   shouldReload?: boolean;
-  /**
-   * When enabling reloadOnVisible, this flag may be used to turn on and off
-   * the reloading. This is useful if your component only needs to reload when
-   * becoming visible during certain states.
-   *
-   * eslint-disable-next-line react/sort-comp
-   */
-  shouldReloadOnVisible?: boolean;
```

Comment:
don't we need this prop anymore?

Reply:
Nope, addressed this in the PR description!

> Removes shouldReloadOnVisible. This was a bit of a holdover from the old AsyncComponent that isn't necessary with the hook, we can just toggle reloadOnVisible.